styles/components/_bankBoard.scss

```diff
@@ -1,28 +1,29 @@
 .bank {
   &__board {
-    display: inline-flex;
-    width: 90vw;
-    align-content: space-around;
+    display: flex;
+    flex-wrap: wrap;
     align-items: center;
-    justify-content: space-around;
-    background-color: wheat;
+    justify-content: space-between;
+    background-color: $color-primary;
+    border-radius: $border-radius;
 
     &__container {
+      display: flex;
+      flex-basis: 50%;
+      justify-content: center;
+      padding-bottom: 1rem;
     }
 
-    &__text {
-      display: inline;
-      font-weight: 700;
+    &__count {
+      padding-top: 1rem;
+      padding-left: 0.5rem;
+      color: $color;
```

Comment:
In trade modal the number by the animals, are supposed to be in tiny circles on the side of the animal. Perhaps we could this with the client to uniform it and prepare it like this also here? Just a suggestion

Reply:
IMO it's clear now because in trade modal those numbers could be clicked, but here - they're just informational.